### PR TITLE
added a route to Event page, created Location component index

### PR DESCRIPTION
**Pull request by AnthoniaOkafor**

### CODE REVIEW

File: ed in both the land [hunk 1]
```diff
@@ -0,0 +1,248 @@
+
```
Comment (meekg33k) [inline ed in both the land]: @AnthoniaOkafor what file is this?

File: src/components/Location/location.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+describe ("Location Component render", () => {
+   it("renders correctly", () => {
+       const wrapper = shallow(<Location name="The name of the event" />);
+       expect(wrapper).toMatchSnapshot();
+    })
+
+    it("renders correctly", () => {
```
Comment (meekg33k) [inline src/components/Location/location.test.js:13]: @AnthoniaOkafor, can we have a test that checks that the correct text renders on the page?

**Review by meekg33k — CHANGES_REQUESTED** [review]

File: src/components/Location/location.test.js (hunk 2 above)
Comment (AnthoniaOkafor) [reply src/components/Location/location.test.js:13]: Ok @meekg33k , I will do just that

**Review by meekg33k — APPROVED** [review]
This looks better @AnthoniaOkafor, good job!

**Pull request merged**